pallets/proposals/src/impls.rs

```diff
@@ -439,6 +439,29 @@ impl<T: Config> Pallet<T> {
             now,
         ));
 
+        //once the voting is complete check if the milestone is eligible for auto approval
+        //Getting the total threshold required for the milestone to be approved based on the raised funds
+        let funding_threshold: BalanceOf<T> = project.raised_funds.saturating_mul(T::PercentRequiredForVoteToPass::get().into())/100u32.into();
+
+        let mut milestone = project.milestones.get_mut(&milestone_key).ok_or(Error::<T>::KeyNotFound)?;
+        //if the yay votes are both greater than the nay votes and the funding threshold then the milestone is approved
+        if updated_vote.yay >= funding_threshold {
+            milestone.is_approved = true;
+             updated_vote = Vote {
```

Comment:
why do you need this? you already updated the vote and have the value in Line 426?

Reply:
Yeah, but the approval part isn't updated there right? we need to do that after checking the threshold no?